esti/hooks.go

```diff
@@ -14,13 +14,16 @@ import (
 
 	"github.com/stretchr/testify/require"
 	"github.com/treeverse/lakefs/pkg/api/apigen"
+	"github.com/treeverse/lakefs/pkg/api/apiutil"
 )
 
 type hooksValidationData struct {
 	data []*webhookEventInfo
 	mu   sync.RWMutex
 }
 
+var expectedRunResultsSize int
```

Comment:
I know that previous runs we just matched hardcoded based on number of tests and now we work relative which is better, not gong to block.

Wanted to suggest to match the action id we produced, this way we not just checking that something new was added.

Reply:
I've consilidated the hooksValidation with the expected result count. Unfortunately the run ID is not returned as part of the webhook data. As a mitigation added a check for the event start time in addition to event type